src/Sinch/SinchClient.cs

```diff
@@ -210,16 +214,17 @@ public ISinchVerificationClient Verification(string appKey, string appSecret)
             {
                 throw new ArgumentNullException(nameof(appKey), "The value should be present");
             }
+
             if (string.IsNullOrEmpty(appSecret))
             {
                 throw new ArgumentNullException(nameof(appSecret), "The value should be present");
             }
-            
+
             var basicAuth = new BasicAuth(appKey, appSecret);
             // TODO: implement application signed authentication, create IHttp just before the request with SignedRequestAuth
             var http = new Http(basicAuth, _httpClient, _loggerFactory?.Create<Http>(), JsonNamingPolicy.CamelCase);
-            return new SinchVerificationClient(appKey, appSecret, 
-                new Uri("https://verification.api.sinch.com/"), 
+            return new SinchVerificationClient(appKey, appSecret,
+                _verificationBaseAddress ?? new Uri("https://verification.api.sinch.com/"),
```

Comment:
Just a quick note.
Personally, having all this kind of constants (URLs for endpoints) centralized in a constant section within the file (or a constant file if this is having sense) facilitate reading sources. 
And if looking for external dependencies it is not "hidden" across files/lines but clearly identified and easier to modify

Reply:
Yeah, easier in case if variable is used more than one time, but when it's the only place then it's makes sense to keep to place of use as near as possible including inlining. But I'll introduce those constants as it makes sense to have them for fast understanding what are these about